### DISCUSSION TITLE
Welcome to Novci Discussions!

### DISCUSSION THREAD

**Discussion by Faza16** [Faza16/Novci#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VampireCake** [comment]
Ovo nam je trebalo

**Comment by Faza16** [comment]
Ovako su i najveci pocinjali

**Comment by VampireCake** [comment]
Moli se mr. Fejzic da ispravi issue koji treba da review-a

**Reply by Faza16** [reply]
Napravljen molim da se gosp Ibisevic integrise i ako ima jos ko zainteresovan kako bi realizacija mogla da krene.

**Reply by VampireCake** [reply]
Na koji način da ga integrišem? Mislim da je najbolje da mi se javi call-om pa da vidim sta cu s njim

**Comment by VampireCake** [comment]
https://www.instagram.com/reel/CzYK6ZwtKjA/?igshid=MTc4MmM1YmI2Ng==
